src/Features/ERDDAP/Platform/Observations/Condition/index.tsx

```diff
@@ -1,55 +1,67 @@
-import * as React from "react"
+/**
+ * Display all timeseries for a specific standard name
+ */
+import React from "react"
```

Comment:
Codacy found an issue: ['React' was used before it was defined.](https://app.codacy.com/manual/gmri/Neracoos-1-Buoy-App/pullRequest?prid=6166252)